pkg/cmd/attestation/verification/attestation.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/sigstore/sigstore-go/pkg/bundle"
 )
 
-var ErrLocalAttestations = errors.New("failed to load local attestations")
+var ErrUnrecognisedBundleExtension = errors.New("bundle file extension not supported, must be json or jsonl")
```

Comment:
@malancas do we need to care about the file extension?

I'm wondering if `GetLocalAttestations()` should just always call `loadBundlesFromJSONLinesFile()`, regardless of the file extension. A `.json` file is really just a one-line `.jsonl` file?

Reply:
I think the call to `loadBundlesFromJSONLinesFile ` would fail if a user provided a JSON bundle that had been pretty printed to the file and therefore took up more than one line. We could proactively remove whitespace for that case if we wanted and then process it line by line.

Reply:
Ah, never mind, a multi-line JSON file makes sense.